### DISCUSSION TITLE
Next track while shuffle play acts different since a couple of weeks

### DISCUSSION THREAD

**Discussion by Fixu69** [rern/rAudio#1632] [Fixed Issues]
First of all, thank you very much for the great app!
A lot has happened here in the last few days, and I'm amazed at how well and quickly the work is being done.

Now my question:
I usually play music in random mode, from a playlist of 500 to about 2000 tracks. If I don't like a track, I click on the next button. In the past, randomly an other track from the list was played, but now the next track is always selected, so it is no longer random. Would it be possible to do it the way it used to be?

Another small issue is that the popup while replacing which is asking >Replace current playlist?< does not close after clicking ok.

Thanks!

**Comment by cYCL157** [comment]
I noticed the same thing this morning.
